deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
+repo=${2-LSSTDESC/start_paper}
+branch=${3-master}
```

Comment:
Gahh, yes you're right, sorry. I think the script should go in doc, and the clean templates in master - that preserves the manual zip download method for starting a paper. Sorry about that!